docs/core/diagnostics/built-in-metrics-system-net.md

```diff
@@ -107,8 +106,7 @@ Available starting in: .NET 8
 | `server.port` | int | Port identifier of the ["URI origin"](https://www.rfc-editor.org/rfc/rfc9110.html#name-uri-origin) HTTP request is sent to. | `80`; `8080`; `443` | If not default (`80` for `http` scheme, `443` for `https`) |
 | `url.scheme` | string | The [URI scheme](https://www.rfc-editor.org/rfc/rfc3986#section-3.1) component identifying the used protocol. | `http`; `https`; `ftp` | Always |
 
-HTTP client request duration measures the time the underlying client handler takes to complete the request up to reading response headers from the network
-stream. It does not include the time spent reading the response body.
+HTTP client request duration measures the time the underlying client handler takes to complete the request. Completing the request includes the time up to reading response headers from the network stream. It doesn't include the time spent reading the response body.
```

Comment:
@noahfalk is this content going to regenerated by YAML so I shouldn't remove the bogus newlines? Or was that just a one time event? If it's a one time even I can remove the newlines, unless you think that generates too much review friction.